### ISSUE TITLE
VisualBasic.BasicLineCommit.CommitOnSave flaky test

### ISSUE THREAD

**Issue by allisonchou** [dotnet/roslyn#46917]
Roslyn.VisualStudio.IntegrationTests.VisualBasic.BasicLineCommit.CommitOnSave

PR: https://github.com/dotnet/roslyn/pull/46905/checks?check_run_id=998181482
Failing build: https://dev.azure.com/dnceng/public/_build/results?buildId=775750&view=ms.vss-test-web.build-test-results-tab&runId=24256418&resultId=100302&paneView=debug

Stack trace:
```
Assert.Contains() Failure\r\nNot found: Sub Main(x   As   Integer)\r\nIn value:  Module Module1\r\n    Sub Main(x   as   Integer)\r\n    End Sub\r\nEnd Module\r\n
at Microsoft.VisualStudio.IntegrationTest.Utilities.OutOfProcess.Editor_OutOfProc.Verifier.TextContains(String expectedText, Boolean assertCaretPosition) in /_/src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/Editor_OutOfProc.Verifier.cs:line 103
   at Roslyn.VisualStudio.IntegrationTests.VisualBasic.BasicLineCommit.CommitOnSave() in /_/src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs:line 96
```

**Comment by allisonchou** [comment]
Hit again here: https://dev.azure.com/dnceng/public/_build/results?buildId=777335&view=ms.vss-test-web.build-test-results-tab&runId=24329434&resultId=100302&paneView=debug

**Comment by sharwell** [comment]
Already fixed by #46811
